src/control/agent/main.go

```diff
@@ -24,41 +24,92 @@
 package main
 
 import (
-	"flag"
+	"github.com/daos-stack/daos/src/control/client"
+	"github.com/daos-stack/daos/src/control/common"
+	"github.com/daos-stack/daos/src/control/drpc"
+	"github.com/daos-stack/daos/src/control/log"
+	"github.com/jessevdk/go-flags"
```

Comment:
Just curious, any specific reason to change this? does the order actually matter?

Reply:
The order does not matter.  The changes were only due to a variety of code edits over time adding and removing packages as I made progress.